### DISCUSSION TITLE
White space on right side of my webpage

### DISCUSSION THREAD

**Discussion by Ramc26** [twbs/bootstrap#39142] [Q&A]
I'm creating a webpage and I added bootstarp class .row and .col-md-6, this created a white space on right side of my-page(if i remove that .col-md-6 class it isn't showing up). I used overflow-x:hidden in css, it solves whitespace on big screens like laptops and desktops, but it is still showing on small screens like mobile phones. 

What Can i do to solve this ?
<img width="496" alt="Screenshot 2023-09-03 at 11 55 16 PM" src="https://github.com/twbs/bootstrap/assets/52733976/7f29a32a-dd84-4371-9012-6fa7e28c77b6">

**Comment by julien-deramond** [comment]
Could you please share your code or a sub-part of it to help you debug it?
